tests/integration/install-upgrade.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+set -exuo pipefail
```

Comment:
So this is currently a copy of the code at https://gitlab.com/bootc-org/tests/bootc-workflow-test right?  Can we include that as a git submodule instead, that gets updated with renovate?

Reply:
Not exactly, I'd like to add more test in the future, like `bootc` arguments tests. So I decided to add a new file here.

Reply:
@henrywang I sent you an invite

Reply:
> New feature or bug fix PR can update and run test in the same PR. And I'd like to add git submodule in bootc-workflow-test repo in the future.

IOW you want to have bootc-workflow-test consume this repository as a git submodule? I think that makes sense...

Reply:
> IOW you want to have bootc-workflow-test consume this repository as a git submodule? I think that makes sense...

Yes, upstream always first.

The workflow I would like to do:
1. most of test code will be saved in upstream repo and QE spend most of their efforts on PRs of upstream repo.
2. bootc-workflow-test `(it will be bootc and bootc image downstream test repo)` will consume upstream repo as `git submodule`. The git update will be triggered by new bootc RHEL `brew` build. That will avoid code gap between upstream`(always latest)` and downstream. I know it can't 100% fix the gap issue, but 
3. then downstream will run daily regular test.